keras_cv/ops/point_cloud.py

```diff
@@ -13,6 +13,16 @@
 # limitations under the License.
 
 import tensorflow as tf
+from tensorflow.python.framework import load_library
+from tensorflow.python.platform import resource_loader
+
+custom_ops = load_library.load_op_library(
```

Comment:
In order to support wheels without custom binaries, I think we'll need to try-catch this so that we don't break at import time if the .so files are missing.

For IoU3D I did this by making IoU3D an object, but IIRC we decided that's probably not the right solution, so maybe we just want to try/except this and raise an Exception when the op is called if we weren't able to load the custom ops at load time.

wdyt?

Reply:
Check https://github.com/tensorflow/addons/pull/869

Reply:
Done.